src/Features/LanguageServer/Protocol/Extensions/ProtocolConversions.cs

```diff
@@ -51,7 +51,7 @@ internal static class ProtocolConversions
             { WellKnownTags.Assembly, LSP.CompletionItemKind.File },
             { WellKnownTags.Class, LSP.CompletionItemKind.Class },
             { WellKnownTags.Constant, LSP.CompletionItemKind.Constant },
-            { WellKnownTags.Delegate, LSP.CompletionItemKind.Method },
+            { WellKnownTags.Delegate, LSP.CompletionItemKind.Delegate },
             { WellKnownTags.Enum, LSP.CompletionItemKind.Enum },
             { WellKnownTags.EnumMember, LSP.CompletionItemKind.EnumMember },
             { WellKnownTags.Event, LSP.CompletionItemKind.Event },
```

Comment:
I think we can also add the other new mapping for `CompletionItemKind.ExtensionMethod` on the line below this one.

I'll leave it up to you if you want to add in this PR, or I can do it later.

Reply:
Yeah, I messed that one